pymc_experimental/distributions/mvncdf.py

```diff
@@ -63,3 +66,10 @@ def R_op(self, inputs, eval_points):
         if eval_points[0] is None:
             return eval_points
         return self.grad(inputs, eval_points)
+
+    
+
+    @_logcdf.register(MvNormalRV)
+    def mvnormal_logcdf(value, mu, cov):
+
+        return pm.logcdf(MvNormal.dist(mu, cov), value)
```

Comment:
Should be something like


```suggestion
mvncdf = Mvncdf()

@_logcdf.register(MvNormalRV)
def mvnormal_logcdf(value, mu, cov):
    return mvncdf(value, mu, cov)
```

Reply:
Which you can then test is working by calling `pm.logcdf(MvNormal.dist(mu, cov), value)`

Reply:
Is there any documentation which shows me how to do this? I tried searching for dispatch function but I have no idea how it works, and what it does. So it's quite tricky for me to know how to fix this.

Reply:
No documentation AFAIK, but you can see how we do it for some custom distributions: https://github.com/pymc-devs/pymc/blob/9b9826c586f04367b5027a0e472122ebcc636139/pymc/distributions/mixture.py#L359-L383

The idea is that the function should return an Aesara expression, which in your case is just the new `Op` you created.